htmlreport/cppcheck-htmlreport

```diff
@@ -399,20 +410,27 @@ def html_escape(text):
 
 def filter_button(enabled_filters, id, function):
     enabled = enabled_filters.get(id, False)
+    if not id:
+        id = 'None'
     return '\n        <label class="checkBtn%s"><input type="checkbox" onclick="%s(this)" id="%s"%s/>%s</label>'\
           % (' disabled' if not enabled else '', function, id, 'checked' if enabled else 'disabled', id)
 
 def filter_bar(enabled):
+    severity_bar = ''.join([filter_button(enabled, severity, 'toggleSeverity') for severity in ['error', 'warning', 'portability', 'performance', 'style', 'information']]) + '\n        | '
+    classification_bar = ''.join([filter_button(enabled, _class, 'toggleClassification') for _class in ['Required', 'Mandatory', 'Advisory', 'Document', '']]) + '\n        | '
```

Comment:
Can classification be updated dynamically somehow.

Misra C 2025 also has the classification Disapplied.

Cert has L1,L2,L3 classifications.

The order is off a little also. It should say `Mandatory, Required, Advisory, Document, Disapplied`. Mandatory is highest prio.